lib/housekeeping.js

```diff
@@ -262,6 +296,7 @@ class HouseKeeper {
 
 
       outcomeInfo[region] = {
+        volumes: totalVolumes,
```

Comment:
That's a good idea. `outcomeInfo` appears to only be for the "bad" outcomes (missing/extraneous request and instances, and zombie IDs). `outcomeInfo` would contain way too much info if we include the "good" volumes.

Reply:
If we put it in the `state` object, we probably won't do it this PR because we'll need to create an `upsertVolumes()` so I'll just remove the line `volumes: totalVolumes;`.